gulpfile.js

```diff
@@ -8,12 +8,22 @@ const sass = require('gulp-sass');
 const filter = require('gulp-filter')
 const minimist = require('minimist');
 const del = require('del');
+const gulpAmpValidator = require('gulp-amphtml-validator');
 const bs = require('browser-sync').create();
 const reload = bs.reload;
+const jsdom = require('jsdom');
+const through = require('through2');
+
+const { JSDOM } = jsdom;
+const AMP_BASE_URL_ELEMENT = '<script async src="https://cdn.ampproject.org/v0.js"></script>';
+const AMP_PLACEHOLDER = '${ampjs}';
+
+const AMP_EXCLUDED_TAGS = new Set(['amp-img']);
+const AMP_REMAPPED_TAGS = {'amp-state': 'amp-bind'};
```

Comment:
Can you include a comment explaining AMP_EXCLUDED_TAGS and AMP_REMAPPED_TAGS?

Also... hmmm.. I'm realizing there are cases where <amp-bind> is used without <amp-state>.  See the first example ("A simple example") here:  https://www.ampproject.org/docs/reference/components/amp-bind

In such cases, the AMP state initializes to "null".  Can we gracefully handle such cases?

Reply:
Following discussion I've added handling for this case.

I've also moved the importer to a separate file, as the logic is polluting the gulpfile a bit so it deserves its own script.